Servers/Administration/OJS.Servers.Administration/Controllers/ContestsController.cs

```diff
@@ -96,6 +96,18 @@ public IActionResult Problems([FromQuery] IDictionary<string, string> complexId)
             AsyncEntityValidators
             => this.contestValidatorsFactory.GetAsyncValidators();
 
+        protected override async Task<IEnumerable<FormControlViewModel>> GenerateFormControlsAsync(Contest entity,
+            EntityAction action, IDictionary<string, string> entityDict,
+            IDictionary<string, Expression<Func<object, bool>>> complexOptionFilters)
+        {
+            var formControls = await base.GenerateFormControlsAsync(entity, action, entityDict, complexOptionFilters)
+                .ToListAsync();
+
+            this.AddEmptyDefaultCategory(formControls);
+
+            return formControls;
+        }
```

Comment:
The `AddEmptyDefaultCategory` can become `PrependDefaultCategory` and returns the collection. Then the code will become:
```
protected override async Task<IEnumerable<FormControlViewModel>> GenerateFormControlsAsync(Contest entity,
            EntityAction action, IDictionary<string, string> entityDict,
            IDictionary<string, Expression<Func<object, bool>>> complexOptionFilters)
  => this.PrependDefaultCategory(
       await base.GenerateFormControlsAsync(entity, action, entityDict, complexOptionFilters)
                .ToListAsync());
```